interfaces/cython/cantera/composite.py

```diff
@@ -655,6 +656,33 @@ def __call__(self, *species):
         return SolutionArray(self._phase[species], states=self._states,
                              extra=self._extra)
 
+    def __len__(self) -> int:
+        return self._shape[0]
+
+    @property
+    def ndim(self) -> int:
+        """The number of dimensions in the SolutionArray.
+
+        :return: `int`
```

Comment:
Is the `:return:` needed if PEP 484 is used? It was my impression that sphinx is smart enough to deduce?

Reply:
I dunno. I'll figure it out 😄  I guess I can just drop the `:return:` annotation since the docstring is clear enough.